Classes/Importer/AbstractImporter.php

```diff
@@ -316,12 +332,28 @@ protected function processBatch(NodeTemplate $nodeTemplate = null)
         $records = $this->preProcessing($records);
 
         array_walk($records, function ($data) use ($nodeTemplate) {
+            if (!\is_array($data)) {
```

Comment:
Basically if you don't do it you bypass the opcache, because PHP need to check if the function is local (in the current namespace) or global (from the core).